src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -732,6 +745,29 @@ public ResponseEntity<Mono<Void>> updateLoadModification(@PathVariable("studyUui
                 .then(studyService.updateEquipmentModification(studyUuid, modifyLoadAttributes, ModificationType.LOAD_MODIFICATION, nodeUuid, modificationUuid)));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{modificationUuid}/{typeModification}")
```

Comment:
I would reverse {modificationUuid} and {typeModification} in the path to fit better the route below

Reply:
I agree @niconoir what do you think ?

Reply:
I agree it would be better. There is also the possibility to add a RequestParam containing the modification type. I don't know which solution would be the best.

Reply:
done